flow-typed/redux.js

```diff
@@ -0,0 +1,54 @@
+declare module 'redux' {
```

Comment:
Should this use/replace https://github.com/flowtype/flow-typed/tree/master/definitions/npm/redux_v3.x.x?